File: app/src/main/java/no/joharei/flixr/mainpage/models/Contacts.java
package no.joharei.flixr.mainpage.models;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class Contacts {

    private int page;
    private int pages;
    private int perPage;
    private int perpage;
    private int total;
    @SerializedName("contact")
    private List<Contact> contacts = new ArrayList<>();

    public List<Contact> getContacts() {
        return contacts;
    }
}

File: app/src/main/java/no/joharei/flixr/mainpage/models/Contacts.kt
package no.joharei.flixr.mainpage.models

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class ContactsResponse(val contacts: Contacts)

@JsonClass(generateAdapter = true)
data class Contacts(
    val page: Int,
    val pages: Int,
    val per_page: Int,
    val total: Int,
    @Json(name = "contact")
    val contacts: List<Contact>
)
